### ISSUE TITLE
Cannot instantiate the Volume Lookup cache

### ISSUE THREAD

**Issue by hickeng** [vmware/vic#2127]
Deployed to VC with NAS storage with the following (which passed validation with no issues):
```
./vic-machine-linux create '-target=...' -image-store=datastore1/localesx/images -compute-resource=/ --name=ghicken-test --volume-store=FC-shared01/volumes:default --bridge-network=Bonneville --external-network=DPortGroup
```

Portlayer failed to come up:
```
time="2016-08-24T22:44:48Z" level=info msg="Adding volume store default ([FC-shared01] volumes)"
time="2016-08-24T22:44:48Z" level=info msg="Creating directory [FC-shared01] volumes/VIC/volumes"
time="2016-08-24T22:44:48Z" level=info msg="Refreshing volume cache."
time="2016-08-24T22:44:48Z" level=panic msg="Cannot instantiate the Volume Lookup cache: error listing vols: Invalid datastore path '[FC-shared01] volumes/VIC/volumes'."
panic: (*logrus.Entry) (0xfcf7e0,0xc420a43140)

goroutine 1 [running]:
panic(0xfcf7e0, 0xc420a43140)
        /usr/local/go/src/runtime/panic.go:500 +0x1a1
github.com/vmware/vic/vendor/github.com/Sirupsen/logrus.Entry.log(0xc420275dc0, 0xc420a34810, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0xc42084f800, 0xc4203c2480, ...)
        /C/Users/georg/Documents/VMware/VIC/src/github.com/vmware/vic/vendor/github.com/Sirupsen/logrus/entry.go:113 +0x3b7
github.com/vmware/vic/vendor/github.com/Sirupsen/logrus.(*Entry).Panic(0xc42084f978, 0xc42084f918, 0x1, 0x1)
        /C/Users/georg/Documents/VMware/VIC/src/github.com/vmware/vic/vendor/github.com/Sirupsen/logrus/entry.go:158 +0x111
github.com/vmware/vic/vendor/github.com/Sirupsen/logrus.(*Entry).Panicf(0xc42084f978, 0x1026bb4, 0x2e, 0xc42084fb68, 0x1, 0x1)
        /C/Users/georg/Documents/VMware/VIC/src/github.com/vmware/vic/vendor/github.com/Sirupsen/logrus/entry.go:206 +0x101
github.com/vmware/vic/vendor/github.com/Sirupsen/logrus.(*Logger).Panicf(0xc420275dc0, 0x1026bb4, 0x2e, 0xc42084fb68, 0x1, 0x1)
        /C/Users/georg/Documents/VMware/VIC/src/github.com/vmware/vic/vendor/github.com/Sirupsen/logrus/logger.go:116 +0xe7
github.com/vmware/vic/vendor/github.com/Sirupsen/logrus.Panicf(0x1026bb4, 0x2e, 0xc42084fb68, 0x1, 0x1)
        /C/Users/georg/Documents/VMware/VIC/src/github.com/vmware/vic/vendor/github.com/Sirupsen/logrus/exported.go:147 +0x5f
github.com/vmware/vic/lib/apiservers/portlayer/restapi/handlers.(*StorageHandlersImpl).Configure(0x1978f60, 0xc420153500, 0xc42001cb40)
        /C/Users/georg/Documents/VMware/VIC/src/github.com/vmware/vic/lib/apiservers/portlayer/restapi/handlers/storage_handlers.go:110 +0xf5c
github.com/vmware/vic/lib/apiservers/portlayer/restapi.configureAPI(0xc420153500, 0x0, 0x0)
        /C/Users/georg/Documents/VMware/VIC/src/github.com/vmware/vic/lib/apiservers/portlayer/restapi/configure_port_layer.go:105 +0x3b8
github.com/vmware/vic/lib/apiservers/portlayer/restapi.(*Server).ConfigureAPI(0xc42036d0e0)
        /C/Users/georg/Documents/VMware/VIC/src/github.com/vmware/vic/lib/apiservers/portlayer/restapi/server.go:31 +0x3e
main.main()
        /C/Users/georg/Documents/VMware/VIC/src/github.com/vmware/vic/cmd/port-layer-server/main.go:64 +0x2d6
time="2016-08-24T22:44:50Z" level=info msg="Launching portlayer server server on 127.0.0.1:6063"
```

This works fine when I specify the same datastore for image-store and for volume-store.
It's worth noting that each host in the cluster has a `datastore1`

**Comment by hickeng** [comment]
Added @gigawhitlocks as well because I'm not sure if this should be a functional setup and is failing (one for @matthewavery), or if it's a case that needs to be rejected in validation.

**Comment by matthewavery** [comment]
@hickeng I believe this should be a validator thing. We have been talking about validating that a compute resource is attached to a target volumestore. But also validating that volumestores are valid should be covered in the scope.

**Comment by gigawhitlocks** [comment]
This should be fixed now, needs verification. @hickeng if you want to see if you can still install with those command line arguments, please have a go at it. Validation should now fail if the volume store datastore isn't accessible.

**Comment by gigawhitlocks** [comment]
If not I'll test this tomorrow when I'm back in the office.

**Comment by gigawhitlocks** [comment]
This is almost certainly resolved by #2126 -- @hickeng please let me know if you are still able to get through validation when running `vic-machine` against this target.
